libs/features/georoutingv2/georoutingv2.css

```diff
@@ -0,0 +1,125 @@
+.georouting-wrapper {
+  padding: 56px 24px 40px;
+}
+
+.dialog-modal.locale-modal-v2 {
+  background-image: url('/libs/features/georoutingv2/img/GeoModal_BG_Map_Mobile.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.picker {
+  position: fixed;
+  margin-top: 2px;
+  background: #FFF 0 0 no-repeat padding-box;
+  box-shadow: 0 1px 4px #00000026;
+  border: 1px solid #B1B1B1;
+  border-radius: 4px;
+  opacity: 1;
+  padding-inline-start: 0;
+  min-width: 176px;
+}
+
+.picker li {
+  list-style-type: none
+}
+
+.picker li a {
+  width: calc(100% - 12px);
+  display: block;
+  padding-left: 12px;
+  color: #222;
+  font-size: 14px;
+}
+
+.picker li a:hover {
+  background-color: rgb(248 248 248);
```

Comment:
Nit: The hover color is barely noticeable for me. Let's update to something darker `#E5E5E5` or `rgb(229, 229, 229)`. 

Since consonant or the XD didn't specify this I used the same hover color used for https://spectrum.adobe.com/page/menu/

Reply:
Yeah, good idea - I've updated the color to that 👍